Movies-Selection/Movies-Selection/ViewController/MovieModule/TableViewCell/CustomTableViewCell.swift

```diff
@@ -31,17 +31,17 @@ final class CustomTableViewCell: UITableViewCell {
 
     // MARK: - Iternal Methods
 
-    func configViewElements(mod: MovieTop?, indexPath: IndexPath) {
+    func configViewElements(mod: [DetailsFilm]?, indexPath: IndexPath) {
         let linkImage = "https://image.tmdb.org/t/p/w500"
         guard let unwrapMod = mod else { return }
-        titleLabelForCell.text = unwrapMod.results[indexPath.row]?.title
-        overviewLabelForCell.text = unwrapMod.results[indexPath.row]?.overview
+        titleLabelForCell.text = unwrapMod[indexPath.row].title
+        overviewLabelForCell.text = unwrapMod[indexPath.row].overview
 
-        guard let unwrapVoteAvarage = unwrapMod.results[indexPath.row]?.voteAverage else { return }
+        let unwrapVoteAvarage = unwrapMod[indexPath.row].voteAverage
         voteAverage.text = String(unwrapVoteAvarage)
 
         DispatchQueue.global().async {
-            guard let imageAdress = unwrapMod.results[indexPath.row]?.posterPath else { return }
+            let imageAdress = unwrapMod[indexPath.row].posterPath
             guard let imageURL = URL(string: linkImage + imageAdress) else { return }
             guard let imageData = try? Data(contentsOf: imageURL) else { return }
```

Comment:
```suggestion
            guard let imageURL = URL(string: linkImage + imageAdress) else { return }
            guard let imageData = try? Data(contentsOf: imageURL) else { return }
```
сегодня вынеси это в сетевой слой